ArraysAmbiguousSNPMiscallingMar2019/README.md

```diff
@@ -8,16 +8,13 @@ Example Invocations:
     ./isVcfAffected.sh affected.vcf 
     The file affected.vcf (of array type: 'MEG_AllofUs_20002558X351448_A1') is affected by the Arrays Ambiguous SNP Bug
 
-    ./isVcfAffected.sh affected.vcf.gz 
-    The file affected.vcf.gz (of array type: 'MEG_AllofUs_20002558X351448_A1') is affected by the Arrays Ambiguous SNP Bug
+    ./isVcfAffected.sh unaffected.vcf.gz 
+    The file unaffected.vcf.gz is NOT affected by the Arrays Ambiguous SNP Bug
```

Comment:
I believe it DOES emit the string '(of array type: foo)' here.

Reply:
i copied this from my console:

```
$ ./isVcfAffected.sh affected.vcf.gz 
The file affected.vcf.gz (of array type: 'MEG_AllofUs_20002558X351448_A1') is affected by the Arrays Ambiguous SNP Bug

$ ./isVcfAffected.sh unaffected.vcf.gz 
The file unaffected.vcf.gz is NOT affected by the Arrays Ambiguous SNP Bug
```

Reply:
feel free to fix the script though, if you want it to be different... :-)